frontend/src/components/Breadcrumbs/makeCrumbs.ts

```diff
@@ -0,0 +1,28 @@
+import { CrumbProperty } from './types';
+
+export const makeCrumbs = (pathname: string, labels: string): CrumbProperty[] => {
```

Comment:
Could you add a comment explaining this function

Reply:
done